database/migrations/2023_11_24_113459_create_email_requests_table.php

```diff
@@ -19,16 +19,16 @@ public function up(): void
             $table->id();
             $table->string('vendor_name')->nullable();
             $table->decimal('vendor_document_amount', 8, 2)->nullable();
-            $table->string('vendor_document_reference')->nullable();
+            $table->string('vendor_document_reference')->nullable()->unique();
             $table->date('vendor_document_date')->nullable();
-            $table->string('vendor_document_filename')->unique();
+            $table->string('vendor_document_filename')->nullable()->unique();
```

Comment:
Isn't this marked as required in Nova? If so, why not required here?

Reply:
It is required if creating manually. If creating via email, then the record gets created first, then the ID is used as part of the filename.